includes/class-coblocks-site-design.php

```diff
@@ -99,7 +99,9 @@ function() {
 	 * @return boolean
 	 */
 	public static function short_circuit_check() {
-		return 'go' !== get_stylesheet();
+		return 'go' !== get_stylesheet() ||
+			// Temporary logic to short-circuit if GSE active.
+			! ( function_exists( '\Go\Core\get_design_style' ) );
```

Comment:
The main reason for this is the testing of GSE. This will be removed, once we merge in the Site Design mods.